### DISCUSSION TITLE
New 5G Attacks

### DISCUSSION THREAD

**Discussion by mchristopoulou** [core-ncsrd/.github#12] [Ideas]
**Objective:** Populate the attack dataset with more attacks against the 5G network.

Examine the attacks on the Word document listed here: https://iptademokritosgr.sharepoint.com/sites/IIT-CORE-RG/_layouts/15/doc.aspx?sourcedoc={38f296cd-f347-4452-a7d9-0805f7a370e1}&action=edit

Check additional attacks from other projects that could fit it.

Open5GS needs to be installed alongside Amarisoft, described here: https://tech-academy.amarisoft.com/MME_Open5GS.html

**Contributors (?)**
@TheReaperGR : performing the attacks
Someone else to install open5gs alongside Amarisoft?
@mchristopoulou 
@vasalosi (to check potential attack scenarios from MARE to also include here?)
@xilouris

**Comment by vasalosi** [comment]
Adding a repo for traffic generation in 5G networks.
5G Traffic Connector is a software created for the generation of both signal and user traffic to be sent to a 5G network core. It is based on implementations from the [**Free5GC**](https://www.free5gc.org/) project and is distributed under an Apache 2.0 license.

The testbed is created using **Open5GS** 
https://github.com/UPM-RSTI/STGUTG

**Reply by mchristopoulou** [reply]
https://zenodo.org/records/8111592
